### ISSUE TITLE
object has no attribute 'PROTOCOL_SSLv3'

### ISSUE THREAD

**Issue by yawanawa** [rupert/pandora-to-google-music#1]
```
(venv) [me@pc ~]$ python2 pandora_to_google_music.py
Traceback (most recent call last):
  File "pandora_to_google_music.py", line 8, in <module>
    import requests
  File "/usr/lib/python2.7/site-packages/requests/__init__.py", line 53, in <module>
    from .packages.urllib3.contrib import pyopenssl
  File "/usr/lib/python2.7/site-packages/requests/packages/urllib3/contrib/pyopenssl.py", line 73, in <module>
    ssl.PROTOCOL_SSLv3: OpenSSL.SSL.SSLv3_METHOD,
AttributeError: 'module' object has no attribute 'PROTOCOL_SSLv3'
```

**Comment by rupert** [comment]
I think upgrading the `requests` library to a newer version should fix this, try:

```
source venv/bin/activate
pip install --upgrade requests
```

**Comment by rupert** [comment]
Thanks for reporting this issue, it's fixed in master (#3).
